cartography/data/jobs/cleanup/gcp_dns_cleanup.json

```diff
@@ -0,0 +1,29 @@
+{
+  "statements": [
+    {
+      "query": "MATCH (n:GCPDNSZone) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
+      "iterative": true,
+      "iterationsize": 100,
+      "__comment__": "Delete GCP DNS Zones that no longer exist and detach them from all previously connected nodes."
+    },
+    {
+      "query": "MATCH (:GCPDNSZone)<-[r:RESOURCE]-(:GCPProject) WHERE r.lastupdated <> {UPDATE_TAG} WITH r LIMIT {LIMIT_SIZE} DELETE (r) return COUNT(*) as TotalCompleted",
```

Comment:
Sorry to move the goal posts yet again, but do you think it'd be possible to incorporate the changes from @lgomezma's latest PR? https://github.com/lyft/cartography/pull/477